policyPrivacy.html

```diff
@@ -0,0 +1,678 @@
+        subject is a client of the controller (Recital 47 Sentence 2 GDPR)
+      </p>
+      <p>
+        9. The legitimate interests pursued by the controller or by a third
+        party Where the processing of personal data is based on Article 6(1)
+        lit. f GDPR our legitimate interest is to carry out our business in
+        favor of the well-being of all our employees and the shareholders.
+      </p>
+      <p>
+        10. Period for which the personal data will be stored The criteria used
+        to determine the period of storage of personal data is the respective
+        statutory retention period. After expiration of that period, the
+        corresponding data is routinely deleted, as long as it is no longer
+        necessary for the fulfillment of the contract or the initiation of a
+        contract.
+      </p>
+      <p>
+        11. Provision of personal data as statutory or contractual requirement;
+        Requirement necessary to enter into a contract; Obligation of the data
+        subject to provide the personal data; possible consequences of failure
+        to provide such data We clarify that the provision of personal data is
+        partly required by law (e.g. tax regulations) or can also result from
+        contractual provisions (e.g. information on the contractual partner).
+        Sometimes it may be necessary to conclude a contract that the data
+        subject provides us with personal data, which must subsequently be
+        processed by us. The data subject is, for example, obliged to provide us
+        with personal data when our company signs a contract with him or her.
+        The non-provision of the personal data would have the consequence that
+        the contract with the data subject could not be concluded. Before
+        personal data is provided by the data subject, the data subject must
+        contact any employee. The employee clarifies to the data subject whether
+        the provision of the personal data is required by law or contract or is
+        necessary for the conclusion of the contract, whether there is an
+        obligation to provide the personal data and the consequences of
+        non-provision of the personal data.
+      </p>
+      <p>
+        12. Existence of automated decision-making As a responsible company, we
+        do not use automatic decision-making or profiling. Developed by the
+        specialists for LegalTech at Willing & Able that also developed the
+        system for gdpr awareness. The legal texts contained in our privacy
+        policy generator have been provided and published by Prof. Dr. h.c.
+        Heiko Jonny Maniero from the German Association for Data Protection and
+        Christian Solmecke from WBS law.
+      </p>
+    </main>
+    <div class="hidden md:flex justify-center items-center pt-20 pb-20">
+      <img class="w-1/2" src="/src/assets/redHero.svg" alt="" />
+    </div>
```

Comment:
Empty `alt` attributes.

```suggestion
    <div class="hidden md:flex justify-center items-center pt-20 pb-20">
      <img class="w-1/2" src="/src/assets/redHero.svg" alt="empty alt attribute" />
    </div>
```